src/sentry/utils/json.py

```diff
@@ -63,6 +63,11 @@ def better_default_encoder(o):
 )
 
 
+def dump(value, fp, **kwargs):
```

Comment:
`**kwargs` aren't used here, seems like it'd be confusing just to drop them silently.